cypress-io/cypress
# fix: handleInvalidTarget breaks the removeAttribute function (#26206)

- Closes #26206

### Additional details


#### Why was this change necessary?

If `handleInvalidTarget`  has run on an `A` (anchor) element (or other target element) no other attribute can be removed except `target`.

### Steps to test


If you have have the following html:

```html
<a id="foo" data-bar="baz"></a>
```

and you have a test with the following:

```JavaScript
cy.get('#foo').click();
```

and then you try to run from the test browser's console the following:

```JavaScript
document.getElementById('foo').removeAttribute('data-bar')
```

the removal of the attribute `data-bar` doesn't work (and I would expect it to).

This PR fixes this. 

### How has the user experience changed?


### PR Tasks
